scripts/fix_linting_issues.py

```
@@ -0,0 +1,207 @@
+        for i, line in enumerate(lines, 1):
+            # Fix visually indented lines with same indent as next logical line
+            # This is a simplified fix - in practice, you'd need more context
+            if i < len(lines) - 1:
+                current_indent = len(line) - len(line.lstrip())
+                next_line = lines[i]
+                next_indent = len(next_line) - len(next_line.lstrip())
+
+                # If current line is continuation and next line has same indent
+                if (line.strip().endswith('and') or line.strip().endswith('or')) and current_indent == next_indent:
+                    # Add proper indentation for continuation
+                    line = ' ' * (current_indent + 4) + line.strip()
```

Ruff (0.12.2)

`for` loop variable `line` overwritten by assignment target

(PLW2901)